routes/chat.js

```diff
@@ -5,6 +5,8 @@ const { ObjectId } = require('mongodb');
 const { User } = require('../models/user');
 const { Chat } = require('../models/chat');
 const { Message, MessageType } = require('../models/message'); // MessageType is used for verification
+const io = require('../serverSocket').io;
```

Comment:
really bad style to import io without incapsulating its functionality in methods.